SocialFilter/sections/specific.txt

```diff
@@ -61,6 +61,7 @@ ukranews.com,everydayhealth.com,grid.id,motorplus-online.com,gridoto.com,bolaspo
 !
+newstomato.com##.vi_remot img[src$="-share.png"]
```

Comment:
Your rule is ok, but as I can see, you can just block the request `||newstomato.com/images/icon/ic-share.png`.

Reply:
OK.